--- parseBlockInfo.js.org       2022-05-13 09:50:59.213476963 +0800
+++ parseBlockInfo.js   2022-05-13 09:51:18.049493647 +0800
@@ -3,10 +3,19 @@
 exports.parseBlockInfo = void 0;
 const block_attributes_1 = require("../block-attributes");
 const parseBlockInfo = (raw = "") => {
+    let initRaw = raw.trim();
+    if(raw.indexOf("{") !== -1 && raw.indexOf("mermaid") === -1) {
+        initRaw =  raw.replace(
+            /^\{\.(.*?)\s/,
+            function (match, p1, offset, str) {
+              return `${p1}${str}`;
+            }
+        )
+    }
     let language;
     let attributesAsString;
     let attributes;
-    const trimmedParams = raw.trim();
+    const trimmedParams = initRaw;
     const match = trimmedParams.indexOf("{") !== -1
         ? trimmedParams.match(/^([^\s{]*)\s*\{(.*?)\}/)
         : trimmedParams.match(/^([^\s]+)\s+(.+?)$/);

